client-js/app/elements/labrad-grapher.ts

```diff
@@ -46,6 +51,8 @@ export class LabradGrapher extends polymer.Base {
   @property({type: String})
   connectionError: string;
 
+  selectedThrottleTimeout: any = null;
```

Comment:
The standard library `setTimeout` function just returns `number` (https://github.com/Microsoft/TypeScript/blob/master/lib/lib.d.ts#L18070).